pkg/controller/core.oam.dev/v1alpha2/application/assemble/testdata/apprevision.yaml

```diff
@@ -279,21 +277,48 @@ spec:
               v\n\t\t\t\t\t\t}\n\t\t\t\t\t},\n\t\t\t\t]\n\t\t\t}\n\t\t}]\n\t}\n}\n\nparameter:
               {\n\t// +usage=Specify the domain you want to expose\n\tdomain: string\n\n\t//
               +usage=Specify the mapping relationship between the http path and the
-              workload port\n\thttp: [string[]: int\n}\n"
+              workload port\n\thttp: [string]: int\n}\n"
         status:
           customStatus: |-
             let igs = context.outputs.ingress.status.loadBalancer.ingress
             if igs == _|_ {
               message: "No loadBalancer found, visiting by using 'vela port-forward " + context.appName + " --route'\n"
             }
             if len(igs) > 0 {
-              if igs[0[].ip != _|_ {
-                message: "Visiting URL: " + context.outputs.ingress.spec.rules[0[].host + ", IP: " + igs[0[].ip
+              if igs[0].ip != _|_ {
+                message: "Visiting URL: " + context.outputs.ingress.spec.rules[0].host + ", IP: " + igs[0].ip
               }
-              if igs[0[].ip == _|_ {
```

Comment:
does it really work before?

Reply:
It has no effect on assemble test cases. Just keep a complete app revision data here.